### PR TITLE
Branching conventions diagram

### PR DESCRIPTION
**Pull request by ettore**
I put this diagram under a new `documentation` folder, let me know if you prefer it elsewhere. Or if we think that it should not live here.

Also let me know if I forgot any edge cases, or anything else that you don't agree with. This is very much something we should all be comfortable in understanding and using.

Might be easier to look at the diagram from here: https://raw.githubusercontent.com/NYPL/ereading-clients/chore/branching-conventions/documentation/branching-conventions.svg

### CODE REVIEW

**Review by samanthaandrews — APPROVED** [review]
Thank you for making this. This fits my understanding of the process as we spoke about it last week. I'm currently writing tickets to set up GH Actions to ensure the OE web release process is up-to-date with this diagram, so I will be referencing it there.

**Comment by kristojorg** [comment]
I love this. The one thing I'd like to add is putting a section in the top level README and rendering this in there. That I think will make it more easily discoverable to devs working on the project

**Comment by ettore** [comment]
> I love this. The one thing I'd like to add is putting a section in the top level README and rendering this in there. That I think will make it more easily discoverable to devs working on the project

sure, good idea, I can do that.

File: README.md [hunk 1]
```diff
@@ -111,12 +111,18 @@ A little history (as far as I understand it). One of the first monorepo tools wa
 - Jest projects only kinda work the way you expect. Their docs say that when using projects the root `jest.config.ts` will be merged with project-specific `jest.config.ts` file while running tests for a workspace. This does not appear to be entirely true, however. Only certain GlobalConfig options are merged. Options like `transform` are not and have to be set on a per-project basis.
 - While NextJS will properly resolve ES Modules, Jest still requires some help. In the package.json `exports` field, we need to specify a `require` property in addition to the `types` and `import` properties used to resolve the ES Module. It can point to the same TS file. See `packages/opds` for an example.
 
-### Building
+### Jest Tests
 
-### Hybrid Apps
```
Comment (ettore) [inline README.md:116]: I realized this "Hybrid Apps" section was kinda misplaced. Since it was just a link, I  moved it to the top

**Comment by kristojorg** [comment]
@ettore I tried to add a white background to the image using CSS so it displays legibly when github is in dark mode... but I just learned that github strips custom CSS from README files. I'm wondering if you can add a white background to the image?

Also, I think the section on naming could be a bit clearer. I think stating the typical git flow branch names is going to be a bit confusing since we are just going to have one main branch called `main`, but unlike git-flow, this won't represent the production branch. Maybe we should stick to just describing the branch names we have and let readers check out git-flow on their own via the link?

**Comment by ettore** [comment]
> @ettore I tried to add a white background to the image using CSS so it displays legibly when github is in dark mode... but I just learned that github strips custom CSS from README files. I'm wondering if you can add a white background to the image?
> 
> Also, I think the section on naming could be a bit clearer. I think stating the typical git flow branch names is going to be a bit confusing since we are just going to have one main branch called `main`, but unlike git-flow, this won't represent the production branch. Maybe we should stick to just describing the branch names we have and let readers check out git-flow on their own via the link?

sounds good, I addressed both things, let me know what you guys think.

**Review by kristojorg — APPROVED** [review]
Looks great to me! Thanks for this

**Review by oliviawongnyc — APPROVED** [review]
Thanks, Ettore!

**Review by Toxiapo — APPROVED** [review]
This is great!

**Review by ErnestFan — APPROVED** [review]

**Pull request merged**